src/dryad/format/endf/createProjectileTarget.hpp

```diff
@@ -0,0 +1,45 @@
+#ifndef NJOY_DRYAD_FORMAT_ENDF_PROJECTILETARGET
+#define NJOY_DRYAD_FORMAT_ENDF_PROJECTILETARGET
+
+// system includes
+#include <vector>
+#include <iostream>
+
+// other includes
+#include "tools/Log.hpp"
+#include "dryad/format/endf/createProjectileIdentifier.hpp"
+#include "dryad/format/endf/createTargetIdentifier.hpp"
+#include "dryad/format/endf/createInteractionType.hpp"
+#include "dryad/format/endf/createReactions.hpp"
+#include "dryad/ProjectileTarget.hpp"
+#include "ENDFtk/Material.hpp"
+#include "ENDFtk/tree/Material.hpp"
+
+namespace njoy {
+namespace dryad {
+namespace format {
+namespace endf {
+
+  /**
+   *  @brief Create a ProjectileTarget from an unparsed ENDF material
+   */
+  ProjectileTarget createProjectileTarget( const ENDFtk::tree::Material& material ) {
+
+    auto mf1mt451 = material.section( 1, 451 ).parse< 1, 451 >();
+
+    ParticleID projectile = createProjectileIdentifier( mf1mt451.subLibrary() );
+    ParticleID target = createTargetIdentifier( mf1mt451.ZA(), mf1mt451.excitedLevel() );
```

Comment:
Will there be any exposed interface to disambiguate levels that happen to be metastable at this level of the interface?

Or, rather, how do I know which level is which?

Reply:
I will most likely make the state identifier have an optional metastable state.